swift/Samples/TabBarContainer/DemoApp/Bar/BarWorkflow.swift

```diff
@@ -0,0 +1,88 @@
+/*
+* Copyright 2019 Square Inc.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+import Workflow
+import WorkflowUI
+import ReactiveSwift
+import TabBarContainer
+
+
+// MARK: Input and Output
+
+struct BarWorkflow: Workflow {
+
+    typealias Output = Never
+
+}
+
+
+// MARK: State and Initialization
+
+extension BarWorkflow {
+
+    struct State {
+
+    }
+
+    func makeInitialState() -> BarWorkflow.State {
+        return State()
+    }
+
+    func workflowDidChange(from previousWorkflow: BarWorkflow, state: inout State) {
+
+    }
+}
+
+
+// MARK: Actions
+
+extension BarWorkflow {
+
+    enum Action: WorkflowAction {
+
+        case viewTapped
+
+        typealias WorkflowType = BarWorkflow
```

Comment:
Doesn't look like WorkFlowType is being used. Can we remove it or use it?

Reply:
This is for conforming to `WorkflowAction`
https://github.com/square/workflow/blob/8d879b0ebd58d11d7495dc3dd01901c98df26b5a/swift/Workflow/Sources/WorkflowAction.swift#L20-L23